snuba/settings_docker.py

```diff
@@ -6,10 +6,10 @@
 DEBUG = env('DEBUG', '0').lower() in ('1', 'true')
 
 CLICKHOUSE_SERVER = env('CLICKHOUSE_SERVER', 'localhost:9000')
-CLICKHOUSE_TABLE = env('CLICKHOUSE_TABLE', 'sentry')
-DEFAULT_DIST_TABLE = CLICKHOUSE_TABLE
-DEFAULT_LOCAL_TABLE = CLICKHOUSE_TABLE
 
+DATASETS = {
```

Comment:
@mattrobenolt, I rebuilt snuba with docker and deployed the image I created via "sentry devservices up" and it all seemed fine. Anything else specific I should watch out about the dev environment?

Reply:
Nope.

Reply:
thanks